.ci/test-cluster.yml

```diff
@@ -78,7 +78,7 @@ services:
     container_name: kuzzle_redis
 
   elasticsearch:
-    image: kuzzleio/elasticsearch:7.16.2
+    image: kuzzleio/elasticsearch:7
```

Comment:
I use this version of ES because it is compatible with Mac M1 ? Is this a problem @alexandrebouthinon ?

Reply:
The `7` version is a pointer on the latest version and should always be used to automatically keep our CI up to date

Reply:
@Aschen is right this is the same image with two different tag